openlibrary/coverstore/code.py

```diff
@@ -38,6 +39,13 @@
 )
 app = web.application(urls, locals())
 
+cors_paths = {
+    '/b/',
+    '/w/',
+}
+app.add_processor(CORSProcessor(cors_paths))
```

Comment:
Misunderstood; your original idea was correct :+1:

```suggestion
app.add_processor(CORSProcessor())
```